src/include/OpenImageIO/paramlist.h

```diff
@@ -711,4 +715,19 @@ class OIIO_UTIL_API ParamValueSpan : public cspan<ParamValue> {
 };
 
 
+// Memory tracking functions.
+
+/// heapsize specialization for `ParamValue`
+namespace pvt {
+template<>
+inline size_t
+heapsize<ParamValue>(const ParamValue& pv)
+{
+    return (pv.m_nonlocal && pv.m_copy)
+               ? pv.m_nvalues * static_cast<int>(pv.m_type.size())
+               : 0;
+}
+}  // namespace pvt
```

Comment:
Again, I would move the implementation into paramlist.cpp, only the declaration (not inline) needs to be in the header.